applications/GeoMechanicsApplication/tests/test_pressure_line_element/test_oblique_line_element3D3N/ProjectParameters.json

```diff
@@ -32,7 +32,6 @@
     "block_builder"                      : true,
     "solution_type"                      : "Transient_Groundwater_Flow",
     "scheme_type"                        : "Backward_Euler",
```

Comment:
Already false, but I agree with removing items that have no meaning for the analysis.